### ISSUE TITLE
Add ability to pass in context

### ISSUE THREAD

**Issue by drem-darios** [joshweir/cdk-pretty-diff#14]
I want to deploy a CDK project that consists of multiple stacks in various environments and regions. When doing a `cdk diff`, I use the flags `--context environment=<ENV>` and `--context region=<REGION>` to set the correct context variables.  I'd like to specify the environment and region when using `cdk-pretty-diff` as well. If there is a way to do this with the existing code, maybe we just need an example in the README. I dug through the code a bit and didn't find a straight forward way to doing this.

**Comment by joshweir** [comment]
Good find i'd had no use for cli args so didn't bother until now. 

It's possible to use cli args but I had to make the input args interface available. I've published v2.2.3 to npm which includes this functionality. 

Updated README with example usage, example usage script [here](https://github.com/joshweir/cdk-pretty-diff/blob/master/bin/diff-to-html-with-cli-args.ts)

```
const configProps: ConfigurationProps = {
  commandLineArguments: {
    _: [Command.DIFF],
    context: [
      'foo=bar',
      'hello=world',
    ],
  }
}
const nicerDiffs = await getCustomDiff({ configProps });
```

cheers
